Describe v3.5 tags. (#1145)
livechat/livechat-public-docs: src/pages/messaging/agent-chat-api/v3.5/rtm-pushes/index.mdx

</CodeResponse>

### `tag_created`

> Informs that a tag has been created within a license. The payload contains the full state of the [tag](/management/configuration-api/v3.5/#tags) data structure, including empty fields.

Reviewer: ```suggestion
Informs that a tag was created within a license. The payload contains the full state of the [tag](/management/configuration-api/v3.5/#tags) data structure, including empty fields.
```

<CodeResponse title={'Sample push payload'}>

```json
{
  "name": "docs_feedback",
  "group_ids": [ 0, 42 ],
  "created_at": "2019-12-09T12:01:18.909000Z",
  "author_id": "smith@example.com"
}
```

</CodeResponse>

### `tag_deleted`

> Informs that a tag has been deleted within a license.

Reviewer: ```suggestion
Informs that a tag was deleted within a license.
```

<CodeResponse title={'Sample push payload'}>

```json
{
  "name": "docs_feedback"
}
```

</CodeResponse>

### `tag_updated`

> Informs that a tag has been updated within a license.

Reviewer: ```suggestion
Informs that a tag was updated within a license.
```
